### ISSUE TITLE
Watcher fails without curly brackets around file extension (css) in sftp.json

### ISSUE THREAD

**Issue by davidchoy** [liximomo/vscode-sftp#300]
## Expected Behavior
Watcher should see and upload `style.css` when it is specified in `sftp.json` as such::

```json
{
    "name":"name",
    "protocol":"sftp",
    "host": "host.com",
    "port": 2222,
    "username": "david",
    "password": "verysecretpassword",
    "remotePath":"/wp-content/themes/mytheme",
    "uploadOnSave": true,
    "watcher": {
        "files": "styles/style.css",
        "autoUpload": true,
        "autoDelete": false
    }
}
```

## Actual Behavior
Watcher does not see `style.css`
It does, however, see `style.css` when it is specified as such:

```json
 "files": "styles/style.{css}",
```

Notice the curly braces.

## Steps to Reproduce the Problem

  1. Specify the watch file without curly braces
  1. Modify the file outside the editor (or using a css preproccessor)
  1. Notice the file does not upload in the output pane
  1. Repeat with curly braces around "css" as shown above, and it works

### Extension Logs

This is when it fails:
```
[info] config at c:\Sites\vs-code-showdown {"name":"name","remotePath":"******","uploadOnSave":true,"downloadOnOpen":false,"syncMode":"update","ignore":[],"watcher":{"files":"styles/style.css","autoUpload":true,"autoDelete":false},"concurrency":4,"protocol":"sftp","host":"******","port":2222,"username":"******","password":"******","connectTimeout":10000,"interactiveAuth":false,"sshConfigPath":"~/.ssh/config","secure":false,"passive":false,"context":"c:\\Sites\\vs-code-showdown"}
[info] [file-save] c:\Sites\vs-code-showdown\styles\sections.less
[info] transfer c:\Sites\vs-code-showdown\styles\sections.less
```

This is when it works:
```
{"name":"name","remotePath":"******","uploadOnSave":true,"downloadOnOpen":false,"syncMode":"update","ignore":[],"watcher":{"files":"styles/style.{css}","autoUpload":true,"autoDelete":false},"concurrency":4,"protocol":"sftp","host":"******","port":2222,"username":"******","password":"******","connectTimeout":10000,"interactiveAuth":false,"sshConfigPath":"~/.ssh/config","secure":false,"passive":false,"context":"c:\\Sites\\vs-code-showdown"}
[info] [file-save] c:\Sites\vs-code-showdown\styles\sections.less
[info] [watcher] c:\Sites\vs-code-showdown\styles\style.css updated
[info] transfer c:\Sites\vs-code-showdown\styles\sections.less
[info] transfer c:\Sites\vs-code-showdown\styles\style.css
``` 

## Specifications

  - Version: 1.26.1
  - Platform: Windows 10 Pro

**Comment by superbuggy** [comment]
This did not occur on Mac OS X with identical configuration.

**Comment by liximomo** [comment]
Does it work with "files": "styles\\style.css",

**Comment by liximomo** [comment]
Does it work with `"files": "styles\\style.css"` in window.

**Comment by liximomo** [comment]
@davidchoy This is fixed by vscode in [#57475](https://github.com/Microsoft/vscode/issues/57475)
